sdk/storage/Azure.Storage.Queues/tests/QueueClientTests.cs

```diff
@@ -1977,5 +1977,49 @@ public void CanMockClientConstructors()
             mock = new Mock<QueueClient>(new Uri("https://test/test"), new AzureSasCredential("foo"), new QueueClientOptions()).Object;
             mock = new Mock<QueueClient>(new Uri("https://test/test"), mockTokenCredential, new QueueClientOptions()).Object;
         }
+
+        [Test]
+        [TestCase("")]
+        [TestCase("u")]
+        [TestCase("ra")]
+        [TestCase("raup")]
+        [TestCase("paru")]
+        public void QueueClientPolicyPermissions_checkIfSetCorrectly(string permissions)
```

Comment:
We should make this a recorded test, and call `QueueClient.SetAccessPolicyAsync()` and `QueueClient.GetAccessPolicyAsync()` to ensure we can property round-trip this new enum with the storage service.